### PR TITLE
modified certificate generator for working with test and prod envs

### PR DESCRIPTION
**Pull request by dimahadgi**

### CODE REVIEW

File: src/utils/certificates_generator.py [hunk 1]
```diff
@@ -8,10 +8,20 @@
 from src.utils.api_helper import ApiHelper
 from src.utils.db_postgress_helper import DbConnect
 
+fake = Faker('en_CA')
+C_NAMES = [fake.company() for x in range(0, 5)]
```
Comment (whichfinder) [inline src/utils/certificates_generator.py]: remove from this module. we discussed this

File: src/utils/certificates_generator.py [hunk 2]
```diff
@@ -21,20 +31,19 @@ def __init__(self):
...
-        db_conn = DbConnect()
-        worker_id = db_conn.fetch_all('''select "id" from "worker" where "employerId"=201 and "archived"=false;''')
-        return [x[0] for x in worker_id]
+    def get_workers_id(self):
+        if Config.env == "prod":
+            workers_list = self.api_helper.do_get_request(self.get_id_url).json()
+            return [d.get('id') for d in workers_list]
+        else:
+            db_conn = DbConnect()
+            worker_id = db_conn.fetch_all('''select "id" from "worker" where "employerId"=202 and "archived"=false;''')
```
Comment (whichfinder) [inline src/utils/certificates_generator.py]: remove hardcoded employerID, move it to config

File: src/utils/certificates_generator.py [hunk 3]
```diff
@@ -21,20 +31,19 @@ def __init__(self):
...
+            return [x[0] for x in worker_id]
 
-    def get_workers_id_from_api(self):
-        workers_list = self.api_helper.do_get_request(self.get_id_url).json()
-        return [d.get('id') for d in workers_list]
-
-    def generate_payload(self, worker_id: int) -> dict:
-        body = {'courseName': self.fake.company(),
+    def generate_payload(self, worker_id: int, course_names) -> dict:
```
Comment (whichfinder) [inline src/utils/certificates_generator.py]: set expected values:
course_names: str

File: src/utils/certificates_generator.py [hunk 4]
```diff
@@ -8,10 +8,20 @@
...
+fake = Faker('en_CA')
+C_NAMES = [fake.company() for x in range(0, 5)]
+
+
+def gen_date():
+    expiration_date = "{}-{}-{}T15:15:10.440Z".format(
+        random.randint(2017, 2023),
+        random.choice(["%.2d" % i for i in range(1, 12)]),
+        random.choice(["%.2d" % i for i in range(1, 25)]))
+    return expiration_date
```
Comment (whichfinder) [inline src/utils/certificates_generator.py:17]: no need to implement this
just get current time stamp ( datetime.datetime.now() ), extend it and format it

Comment (whichfinder) [reply src/utils/certificates_generator.py:17]: and use inside generate_payload()

**Review by whichfinder — CHANGES_REQUESTED** [review]

File: src/utils/certificates_generator.py (hunk 1 above)
Comment (dimahadgi) [reply src/utils/certificates_generator.py]: fixed

File: src/utils/certificates_generator.py (hunk 2 above)
Comment (dimahadgi) [reply src/utils/certificates_generator.py]: fixed

File: src/utils/certificates_generator.py (hunk 3 above)
Comment (dimahadgi) [reply src/utils/certificates_generator.py]: fixed

**Review by whichfinder — APPROVED** [review]

**Pull request merged**